cmd/aws-cloud-controller-manager/main.go

```diff
@@ -117,8 +117,13 @@ func main() {
 				os.Exit(1)
 			}
 
+			cloudConfigFile := c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile
```

Comment:
@andrewsykim updated! I think it would be good to warn when the file path is empty, wdyt?

Reply:
I think the warning isn't needed if the process would exit and fail if it was empty. But we would need to double check if that's the case. If `--cloud-config` is optionally the warning is probably not needed either.

Reply:
gotcha, I think the configuration file path is optional by checking the code, so I remove the warning here

Reply:
The examples should probably be updated as well instead of using a hardcoded path https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/sample/basic_main.go#L66